packages/scrollbar/src/index.vue

```diff
@@ -91,14 +104,19 @@ export default defineComponent({
     }
 
     const style = computed(() => {
-      if (Array.isArray(props.wrapStyle)) {
-        return toObject(props.wrapStyle)
+      let style = props.wrapStyle
+      if (Array.isArray(style)) {
+        style = toObject(style)
+        style.height = addUnit(props.height)
+        style.maxHeight = addUnit(props.maxHeight)
+      } else if (typeof style === 'string') {
+        style += addUnit(props.height) ? `height: ${addUnit(props.height)};` : ''
+        style += addUnit(props.maxHeight) ? `max-height: ${addUnit(props.maxHeight)};` : ''
       }
-      return props.wrapStyle
+      return style
     })
 
     onMounted(() => {
-      if (props.native) return
       nextTick(update)
```

Comment:
If we are using `native` scrollbar, what were we updating anyway? Since the `bar` item won't even be rendered, we should exclude this line when native is `true`, something like:
```ts
if (!props.native) {
  nextTick(update)
}
// other things.
```